README.md

```diff
@@ -68,7 +68,14 @@ cd %appdata%\helix
 mklink /D runtime "<helix-repo>\runtime"
 ```
 
-This location can be overridden via the `HELIX_RUNTIME` environment variable.
+The runtime location can be overridden via the `HELIX_RUNTIME` environment variable.
+
+> NOTE: if `HELIX_RUNTIME` is set prior to calling `cargo install --path helix-term`
+> , tree-sitter grammars will be built in `$HELIX_RUNTIME/grammars`.
```

Comment:
The comma breaking over the line is a bit odd and shows up as a space when rendered - can you move this to the first line?

Reply:
Good eye. Should be fixed up in https://github.com/helix-editor/helix/pull/4382/commits/0bc7453ae2e8638fb12144c7c47c30fb9d56a6fc.